### ISSUE TITLE
Notify only one repo

### ISSUE THREAD

**Issue by GTP95** [ghaiklor/telegram-bot-github#130]
Hello everyone, I have a question: it's possible to make this bot send notifications only about updates of a specific repo? My idea is to add this bot to a Telegram group of people working on the same project, so we have everything about the project in one place, but the bot should notfy changes about only that repository and not others. Is this currently possible?

**Comment by ghaiklor** [comment]
@GTP95 nope. This bot tied up to your user account and read all notifications specifically for your account.

**Comment by puribapu9141** [comment]
Sir Where We Can Host This Bot?

**Comment by ghaiklor** [comment]
Closing this one. I designed the bot to reply with all notifications from GitHub as is (using `/notifications` endpoint). It simplifies the development a lot, so that all the logic for notifications sorting\filtering is happening on GitHub side.

If you want to add such an ability, configure your GitHub profile for that. E.g. stop watching the repositories you don't want to get, so `/notifications` endpoint will stop sending me those.
